reviewdog.go

```diff
@@ -172,6 +172,10 @@ func addedDiffLines(filediffs []*diff.FileDiff, strip int) posToAddedLine {
 	return r
 }
 
+func NewDummyFunc() error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewDummyFunc should have comment or be unexported